### PR TITLE
W-10875962: Add Exchange dependency update to workspace validation

### PR DESCRIPTION
**Pull request by fermujica**

### CODE REVIEW

File: modules/ROOT/pages/update-workspace.adoc [hunk 1]
```diff
@@ -71,6 +72,26 @@ Do not to preserve the following declarations:
 * `classLoaderModelLoaderDescriptor`
 * `bundleDescriptorLoader`
 
+[[exchange-dependency-update]]
+== Exchange Update for Workspaces Created With Studio 7.11.x and Earlier Versions
+
+Because Anypoint Studio 7.12 adds support for comprehensive publication of assets to Exchange, your projects must add a dependency to the new version of Anypoint Exchange.
```
Comment (tomasperezc) [inline modules/ROOT/pages/update-workspace.adoc:78]: I'm not sure about using `dependency` here, I would change this to `repository`. Feels more accurate

File: modules/ROOT/pages/update-workspace.adoc [hunk 2]
```diff
@@ -71,6 +72,26 @@ Do not to preserve the following declarations:
...
+
+When you open a project created with earlier versions than 7.12, Studio adds this new dependency to your project pom.xml file:
```
Comment (tomasperezc) [inline modules/ROOT/pages/update-workspace.adoc]: same as comment above

File: modules/ROOT/pages/update-workspace.adoc [hunk 3]
```diff
@@ -71,6 +72,26 @@ Do not to preserve the following declarations:
...
+--
+<repository>
+  <id>anypoint-exchange-v3</id>
+  <name>Anypoint Exchange V3</name>
+  <url>https://maven.anypoint.mulesoft.com/api/v3/maven</url>
+  <layout>default</layout>
+</repository>
+--
+
+You must add this new dependency to take advantage of the latest publication process to Exchange.
```
Comment (tomasperezc) [inline modules/ROOT/pages/update-workspace.adoc]: same as comment above

File: modules/ROOT/pages/update-workspace.adoc [hunk 4]
```diff
@@ -3,21 +3,22 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio helps you to ensure that you are using the latest Mule Maven configuration.
+Anypoint Studio helps you ensure that you are using the latest Mule Maven plugin and that your project points to the latest Anypoint Exchange. When Studio runs in a new workspace for the first time, it validates that every project in your workspace has both dependencies updated.
```
Comment (tomasperezc) [inline modules/ROOT/pages/update-workspace.adoc]: I would change this for something like `it validates that your projects are up to date and ready to use in Anypoint Studio 7.12.0`. I would avoid using `dependency`, so it is not mistaken for the dependency section of the pom.xml, which we are not editing

**Review by tomasperezc — DISMISSED** [review]
Added some comments, but just as suggestions. Feel free to use them or leave the document as it is 🙂

File: modules/ROOT/pages/update-workspace.adoc (hunk 4 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/update-workspace.adoc]: Would it OK to say:
..."it validates that the components of such workspaces are updated and compliant with latest Studio 7.12.0 requirements."

File: modules/ROOT/pages/update-workspace.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/update-workspace.adoc:78]: The possessive seems to be inappropriate here: "...to identify the application's classloader configuration and settings...". It should read "...to identify the application classloader configuration and settings..."

File: modules/ROOT/pages/update-workspace.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/update-workspace.adoc]: Word order: "...a project created with earlier versions than 7.12, Studio..."
It should read: "...a project created with versions earlier than 7.12, Studio"

**Review by Crispy-Salesforce — CHANGES_REQUESTED** [review]
Changes made to PR 308 as requested.
@fermujica, please, can you approve or assign an approver for this PR?
Thanks!

File: modules/ROOT/pages/update-workspace.adoc [hunk 5]
```diff
@@ -3,20 +3,20 @@ ifndef::env-site,env-github[]
...
+Anypoint Studio helps you ensure that you are using the latest Mule Maven plugin and that your project points to the latest Anypoint Exchange. When Studio runs in a new workspace for the first time, it validates that the components of such workspaces are updated and compliant with latest Studio 7.12.0 requirements.
 
 To manually validate your workspace navigate to *Help* > *Validate workspace compatibility*.
 
 image::update-workspaces.png[alt=Dialog box that provides the possibility to select the updates that you want to perform.]
 
 <1> Studio lists all the open projects in your workspace that must update the Mule Maven Plugin version.
-<1> Studio lists all the open projects in your workspace that must add the updated Exchange dependency.
-<1> Selecting *Perform update* enables Studio to update the dependencies in the `pom.xml` file of the projects you selected.
+<1> Studio lists all the open projects in your workspace that must add the updated Exchange repository.
```
Comment (fermujica) [inline modules/ROOT/pages/update-workspace.adoc:13]: Probably "version" here would be more straightforward

> Studio lists all the open projects in your workspace that must add the updated Exchange version.

Comment (Crispy-Salesforce) [reply modules/ROOT/pages/update-workspace.adoc:13]: You are the boss!
"Version" sounds like a great option, so go ahead with it.

File: modules/ROOT/pages/update-workspace.adoc [hunk 6]
```diff
@@ -3,21 +3,22 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio helps you to ensure that you are using the latest Mule Maven configuration.
+Anypoint Studio helps you ensure that you are using the latest Mule Maven plugin and that your project points to the latest Anypoint Exchange. When Studio runs in a new workspace for the first time, it validates that the components of such workspaces are updated and compliant with latest Studio 7.12.0 requirements.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/update-workspace.adoc]: the latest Anypoint Exchange > the latest Anypoint Exchange *version*

**Review by Cristian-Venticinque — APPROVED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]
All changes OK!
Ready to merge.

**Pull request merged**